doc/en/user/source/data/webadmin/workspaces.rst

```diff
@@ -93,71 +87,86 @@ Once enabled clicking on the service link will open the settings page for the se
 
 Clients accessing this workspace as a :ref:`virtual_services` will use the service description provided here.
 
-Security
-^^^^^^^^
+.. _workspace_settings:
 
-The Security tab allows to set data access rules at workspace level.
+Workspace Settings
+^^^^^^^^^^^^^^^^^^
 
-.. note:: For more information on data access rules, please see the section on :ref:`security_webadmin_data`.
+Use :guilabel:`Enabled` checkbox to override the global configuration and contact information for this workspace.
 
-.. figure:: img/data_workspaces_security_edit.png
+.. figure:: img/workspace_settings.png
+   
+   Enable workspace settings to provide default contact information
 
-To create/edit the workspace's data access rules, check/uncheck checkboxes according to the desired role. 
-The Grant access to any role checkbox grant each role for any access mode.
+Organization
+''''''''''''
 
-Isolated Workspaces
-^^^^^^^^^^^^^^^^^^^
+Clients accessing this workspace as a :ref:`virtual_services` will use the organization description provided here:
 
-Isolated workspaces content is only visible and queryable in the context of a virtual service bound to the isolated workspace. This means that isolated workspaces content will not show up in global capabilities documents and global services cannot query isolated workspaces contents. Is worth mentioning that those restrictions don't apply to the REST API.
+* The :guilabel:`Welcome` message is used as an introduction in the welcome page header for this workspace.
 
-A workspace can be made isolated by checking the :guilabel:`Isolated Workspace` checkbox when creating or editing a workspace.
+* The :guilabel:`Organization` name and :guilabel:`Online Resource` are combined to form a organization link in the welcome page header for this workspace.
+
+.. figure:: img/workspace_orgaization.png
+   
+   Workspace Organization
 
-.. figure:: img/isolated_workspace.png
+If this information is not provided the global :ref:`config_contact` page contact organization details are used.
 
-   Making a workspace isolated
+Primary Contact
+'''''''''''''''
 
-An isolated workspace will be able to reuse a namespace already used by another workspace, but its resources (layers, styles, etc ...) can only be retrieved when using that workspace virtual services and will only show up in those virtual services capabilities documents.
+Clients accessing this workspace as a virtual service, or via the welcome page, will use the contact information provided here:
 
-It is only possible to create two or more workspaces with the same namespace in GeoServer if only one of them is non isolated, i.e. isolated workspaces have no restrictions in namespaces usage but two non isolated workspaces can't use the same namespace.
+* The :guilabel:`email` address if provided, will be used as the administrator contact in the welcome page footer for this workspace.
 
-The following situation will be valid:
+.. figure:: img/workspace_contaxt.png
```

Comment:
The file name should have been `workspace_context.png`. However as long as they match....